apps/storybook/src/Surface.stories.tsx

```diff
@@ -39,3 +50,177 @@ export const Basic: Story<SurfaceProps> = ({ elevation }) => {
 };
 
 Basic.args = { elevation: 1 };
+
+export const Custom: Story<SurfaceProps> = (args) => {
+  const { elevation, hasLayout, isPadded } = args;
+  return (
+    <Surface elevation={elevation} hasLayout={hasLayout} {...args}>
```

Comment:
Could you add one example to docs website?

Reply:
This example has been added